src/ch01-00-getting-started.md

```diff
@@ -4,7 +4,7 @@ Starknet is a scalable layer 2 solution on Ethereum. This guide will walk you th
 
 We will use the [Starknet Remix Plugin](https://github.com/NethermindEth/starknet-remix-plugin?tab=readme-ov-file) to compile, deploy, and interact with our smart contract. It's a great tool for getting started with Starknet development because you don't need to install anything on your computer.
```

Comment:
I think we can provide a bit more context about Remix here. I recommend the following changes:

We will use the [Remix IDE](https://remix-ide.readthedocs.io/en/latest/), a no-setup, browser-based smart contract development environment, along with the [Starknet Remix Plugin](https://github.com/NethermindEth/starknet-remix-plugin?tab=readme-ov-file) to compile, deploy, and interact with our smart contract.